samples/memfs/memfs.go

```diff
@@ -157,9 +163,10 @@ func (fs *memFS) getInodeOrDie(id fuseops.InodeID) *inode {
 //
 // LOCKS_REQUIRED(fs.mu)
 func (fs *memFS) allocateInode(
-	attrs fuseops.InodeAttributes) (id fuseops.InodeID, inode *inode) {
+	attrs fuseops.InodeAttributes,
```

Comment:
Here and elsewhere, please don’t add line breaks into function signatures. I know the code currently does that, but it’s not idiomatic Go style and I’d like to get rid of line breaks whenever possible.